lib/assets/chart_cell/main.js

```diff
@@ -260,34 +302,122 @@ export function init(ctx, payload) {
               >{{ optionLabel(modelValue) }}</option>
             </select>
           </div>
-        `
+        `,
       },
-    }
+      Accordion: {
+        data() {
+          return {
+            isOpen: payload.layers.length <= 2,
+          };
+        },
+        props: {
+          hasLayers: {
+            type: Boolean,
+            required: true,
+          },
+        },
+        methods: {
+          toggleAccordion() {
+            this.isOpen = !this.isOpen;
+          },
+        },
+        template: `
+          <div class="layer-wrapper" :class="{'wrapper--closed': !isOpen}">
+            <div
+              class="accordion-control"
+              :aria-expanded="isOpen"
+              :aria-controls="id"
+              :class="{'control--sm': isOpen}"
+            >
+              <span v-show="!isOpen"><slot name="title" /></span>
+              <span></span>
+              <span>
+                <svg
+                  v-show=hasLayers
+                  @click="$emit('removeLayer')"
+                  class="accordion-svg"
```

Comment:
If possible it would be better to have `<button><svg>...</svg></button>`, where svg is just the icon and event/visibility logic is on the button :)